### ISSUE TITLE
Freezes Emacs when inserting yasnippet (sometimes)

### ISSUE THREAD

**Issue by chookity-pokk** [emacs-vs/line-reminder#23]
Steps to reproduce:
Launch Emacs -> open an Org mode file -> try to expand a yasnippet -> Emacs freezes -> in terminal run `pkill -SIGUSR2 emacs` -> in terminal run `pkill emacs` -> Emacs Debugger gives below snippet

So I am not entirely sure how to read the debugger so I put the whole thing in there. It seems to happen about 80% of the time when trying to expand a snippet (only used in Org Mode though). 

I also tried turning off `global-line-reminder-mode` then expanding a snippet, which worked just fine. Then I re-enabled `global-line-reminder-mode` and expanded the same snippet and it froze.

```
Debugger entered--entering a function:
* substitute-command-keys("Quit")
  line-number-at-pos(2901 t)
  line-reminder--line-number-at-pos(2901)
  line-reminder--after-change(2877 2900 0)
  run-hook-with-args(line-reminder--after-change 2877 2900 0)
  #f(compiled-function () #<bytecode -0x1359306ed5e43745>)()
  funcall(#f(compiled-function () #<bytecode -0x1359306ed5e43745>))
  (let nil (funcall '#f(compiled-function () #<bytecode -0x1359306ed5e43745>)))
  eval((let nil (funcall '#f(compiled-function () #<bytecode -0x1359306ed5e43745>))))
  yas--snippet-create("#+BEGIN_SRC $0\n\n#+END_SRC" nil 2877 2877)
  yas-expand-snippet(#s(yas--template :key "insert" :content "#+BEGIN_SRC $0\n\n#+END_SRC" :name "insert" :condition nil :expand-env nil :load-file "/home/hank/.emacs.d/snippets/org-mode/insert.yasni..." :save-file nil :keybinding nil :uuid "insert" :menu-binding-pair ((menu-item "insert" (lambda nil (interactive) (yas--expand-or-visit-from-menu 'org-mode "insert")) :keys "insert =>")) :group nil :perm-group nil :table #s(yas--table :name "org-mode" :hash #<hash-table equal 4/65 0x158f1e15f46b> :uuidhash #<hash-table equal 4/65 0x158f1e15f591> :parents nil :direct-keymap (keymap))) 2877 2883)
  yas--expand-or-prompt-for-template((("insert" . #s(yas--template :key "insert" :content "#+BEGIN_SRC $0\n\n#+END_SRC" :name "insert" :condition nil :expand-env nil :load-file "/home/hank/.emacs.d/snippets/org-mode/insert.yasni..." :save-file nil :keybinding nil :uuid "insert" :menu-binding-pair ((menu-item "insert" (lambda nil (interactive) (yas--expand-or-visit-from-menu ... "insert")) :keys "insert =>")) :group nil :perm-group nil :table #s(yas--table :name "org-mode" :hash #<hash-table equal 4/65 0x158f1e15f46b> :uuidhash #<hash-table equal 4/65 0x158f1e15f591> :parents nil :direct-keymap (keymap))))) 2877 2883)
  yas-expand()
  funcall-interactively(yas-expand)
  call-interactively(yas-expand nil nil)
  command-execute(yas-expand)
```

**Comment by jcs090218** [comment]
I can confirm this issue, thanks for reporting it to us!

I was wondering why the whole time... 😅

**Comment by jcs090218** [comment]
I've opened a fix in #24. Let me know if the issue is addressed!

**Comment by chookity-pokk** [comment]
I just expanded a handful of snippets and it didn't freeze so I think it's fixed! Thank you for the quick fix!
